src/main/java/com/serenitydojo/CartItem.java

```diff
@@ -0,0 +1,34 @@
+package com.serenitydojo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class CartItem {
+
+    private final Fruit fruit;
+    private final double quantity;
+
+    List<CartItem> cartItems = new ArrayList<>();
+
+    public CartItem(Fruit fruit , double quantity){
+        this.fruit =fruit;
+        this.quantity = quantity;
+    }
+
+    public void addCartItem(CartItem item){
+        cartItems.add(item);
+    }
+
```

Comment:
```suggestion
    public CartItem(Fruit fruit , double quantity){
        this.fruit =fruit;
        this.quantity = quantity;
    }
```
unused, remove cartItems